watcher/watcher.go

```diff
@@ -238,7 +238,7 @@ func (wh *WatchHandler) HandleSBOMFilteredEvents(sfEvents <-chan watch.Event, pr
 			continue
 		}
 
-		wlid, ok := obj.ObjectMeta.Annotations[instanceidhandlerv1.WlidAnnotationKey]
+		wlid, ok := obj.ObjectMeta.Annotations[instanceidhandlerv1.WlidMetadataKey]
```

Comment:
?

Reply:
Razi renamed it
https://github.com/kubescape/k8s-interface/pull/45/files